ui/perfherder/shared/Pagination.jsx

```diff
@@ -53,24 +62,28 @@ class PaginationGroup extends React.Component {
             </PaginationLink>
           </PaginationItem>
         ))}
-        {currentPage < count && (
-          <PaginationItem>
-            <PaginationLink
-              className="text-info"
-              next
-              onClick={() => this.navigatePage(currentPage + 1)}
-            />
-          </PaginationItem>
-        )}
-        {lastViewablePage < count && (
-          <PaginationItem className="text-info">
-            <PaginationLink
-              className="text-info"
-              last
-              onClick={() => this.navigatePage(count)}
-            />
-          </PaginationItem>
-        )}
+        <PaginationItem disabled={!nextButtonAvailable}>
+          <PaginationLink
+            className="text-info"
+            next
+            {...(nextButtonAvailable
```

Comment:
Can you tell me how is this check useful?
This looks like a double check for `nextButtonAvailable`.
I tested this locally only with the `onClick` event and it seems to work fine because we have the `disabled` props on `PaginationItem`.

Reply:
Thank you for pointing that out.
I didn't realize `disabled` affects `onClick`.
I confirmed that event handler isn't registered for the button if `disabled` is set.
will remove the branch.